tide_core.install

```diff
@@ -1331,3 +1331,20 @@ function tide_core_update_8054() {
     $config->set('cardinality', -1)->save();
   }
 }
+
+/**
+ * Check if the "tide_cms_support" module is installed
```

Comment:
Update the comment to say - Enable "tide_cms_support" module, if it is not enabled.

Reply:
Resolved